translations.json

```diff
@@ -1,13 +1,14 @@
 {
   "nl": {
     "translation": {
-      "avgDisclaimer": "De gemeente Amsterdam verwerkt uw persoonsgegevens op een zorgvuldige en veilige manier in overeenstemming met de geldende wet- en regelgeving, waaronder de Algemene Verordening Gegevensbescherming. Lees meer hierover op <a href='$t(avgDisclaimerUrl)'>Privacyverklaring Meldingen openbare ruimte en overlast</a>.",
+      "avgDisclaimer": "{{organisationName, capitalize}} verwerkt uw persoonsgegevens op een zorgvuldige en veilige manier in overeenstemming met de geldende wet- en regelgeving, waaronder de Algemene Verordening Gegevensbescherming. Lees meer hierover op <a href='$t(avgDisclaimerUrl)'>Privacyverklaring Meldingen openbare ruimte en overlast</a>.",
```

Comment:
What spelling are we using? British (`organisation`, `capitalise`) or American (`organization`, `capitalize`)? Let's be consistent.

Reply:
I'd prefer American and think that's what's being used predominantly throughout the project already.